lib/matplotlib/colorbar.py

```diff
@@ -839,7 +839,7 @@ def _get_ticker_locator_formatter(self):
                 # default locator:
                 nv = len(self._values)
                 base = 1 + int(nv / 10)
```

Comment:
Should this be moved up a level in the elif after `BoundaryNorm`?  It seems out of place here...

Reply:
indeed.